File: client/mc/common2/src/main/java/calebxzhou/rdi/mc/common2/mcp/RMcpEntityDetailData.java
package calebxzhou.rdi.mc.common2.mcp;

import java.util.Map;

public record RMcpEntityDetailData(
        String dim,
        String uuid,
        String type,
        String name,
        RMcpPosData pos,
        Map<String, Object> runtime,
        Map<String, Object> nbt,
        String snbt
) {
}

File: mc/rmcp/common/src/main/kotlin/calebxzhou/rdi/mc/common2/mcp/RMcpEntityDetailData.kt
package calebxzhou.rdi.mc.common2.mcp

@JvmRecord
data class RMcpEntityDetailData(
    val dim: String,
    val uuid: String,
    val type: String,
    val name: String,
    val pos: REntityPosData,
    val runtime: Map<String, Any?>,
    val snbt: String?
)
